replibyte/src/transformer/keep_first_char.rs

```diff
@@ -0,0 +1,134 @@
+use crate::transformer::Transformer;
+use crate::types::Column;
+
+pub struct KeepFirstCharTransformer {
+    database_name: String,
+    table_name: String,
+    column_name: String,
+}
+
+impl KeepFirstCharTransformer {
+    pub fn new<S>(database_name: S, table_name: S, column_name: S) -> Self
+    where
+        S: Into<String>,
+    {
+        KeepFirstCharTransformer {
+            database_name: database_name.into(),
+            table_name: table_name.into(),
+            column_name: column_name.into(),
+        }
+    }
+}
+
+impl Transformer for KeepFirstCharTransformer {
+    fn id(&self) -> &str {
+        "keep_first_char"
+    }
+
+    fn description(&self) -> &str {
+        "Keep only the first character of the column."
+    }
+
+    fn database_name(&self) -> &str {
+        self.database_name.as_str()
+    }
+
+    fn table_name(&self) -> &str {
+        self.table_name.as_str()
+    }
+
+    fn column_name(&self) -> &str {
+        self.column_name.as_str()
+    }
+
+    fn transform(&self, column: Column) -> Column {
+        match column {
+            Column::NumberValue(column_name, value) => {
+                Column::NumberValue(column_name, get_first_digit(value))
+            }
+            Column::FloatNumberValue(column_name, value) => {
+                Column::FloatNumberValue(column_name, value)
+            }
+            Column::StringValue(column_name, value) => {
+                let new_value = if value == "".to_string() {
+                    value
+                } else {
+                    value.chars().nth(0).unwrap().to_string()
```

Comment:
I'm not sure with this `unwrap` to get the first char. So, I'd like to have some pointers :) I think it's safe to use an unwrap here because I've check that the value is not empty in the if condition.  Thanks for your help.

Reply:
It looks good to me even if I would try to avoid using unwrap just in case of. It looks good here :)

Reply:
OK, thx for your opinion, I've refactored a little bit to handle this possibility.